daemon/algod/api/algod.oas2.json

```diff
@@ -3293,6 +3293,10 @@
         "allow-more-logging": {
           "description": "Lifts limits on log opcode usage during simulation.",
           "type": "boolean"
+        },
+        "allow-extra-budget": {
```

Comment:
Can you be more specific that this is app budget? In the future we may also want to support extra logicsig budget.

Also, IMO the `allow-` prefix implies that the field is a boolean.